.github/scripts/process_commit.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+"""
+This script finds the merger responsible for labeling a PR by a commit SHA. It is used by the workflow in
+'.github/workflows/pr-labels.yml'. If there exists no PR associated with the commit or the PR is properly labeled,
+this script is a no-op.
+
+Note: we ping the merger only, not the reviewers, as the reviewers can sometimes be external to pytorch
+with no labeling responsibility, so we don't want to bother them.
+This script is based on: https://github.com/pytorch/vision/blob/main/.github/process_commit.py
+"""
+
+import sys
+from typing import Any, Optional, Set, Tuple
+
+import requests
+
+# For a PR to be properly labeled it should have one primary label and one secondary label
+
+RELEASE_NOTES_LABELS = {
```

Comment:
alternately is there a to pull out all the `release notes: ...` labels here instead of manually mentioning a list. This would make it easier for people to add more release notes labels just via github (without a codechange) if they want to.

Reply:
Great idea! Let me look into it!